### PR TITLE
add badges

### PR DESCRIPTION
**Pull request by trontrytel**
@charleskawczynski - don't know how to add the correct link to bors badge. Everything else works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+[docs-dev-url]: https://CliMA.github.io/CloudMicrophysics.jl/dev/
+
+[gha-ci-img]: https://github.com/CliMA/CloudMicrophysics.jl/actions/workflows/ci.yml/badge.svg
+[gha-ci-url]: https://github.com/CliMA/CloudMicrophysics.jl/actions/workflows/ci.yml
+
+[codecov-img]: https://codecov.io/gh/CliMA/CloudMicrophysics.jl/branch/main/graph/badge.svg
+[codecov-url]: https://codecov.io/gh/CliMA/CloudMicrophysics.jl
+
+[bors-img]: https://bors.tech/images/badge_small.svg
+[bors-url]: https://app.bors.tech/repositories/TODO
```
Comment (charleskawczynski) [inline README.md:25]:
```suggestion
[bors-url]: https://app.bors.tech/repositories/35474
```

**Review by charleskawczynski — APPROVED** [review]
Looks good!

**Comment by trontrytel** [comment]
@JuliaRegistrator register

**Pull request merged**